client/packages/common/src/ui/icons/MSupplyGuy.tsx

```diff
@@ -73,9 +73,7 @@ export const AnimatedMSupplyGuy = styled(SvgGuy)(({ theme, size }) => {
     }),
     '&:hover': {
       animation:
-        size === 'large'
-          ? `${spin} 1s infinite ease`
-          : `${otherSpin} 1s infinite ease`,
+        size === 'large' ? `${spin} 1s  ease` : `${otherSpin} 1s 1 ease`,
```

Comment:
offroad: the infinite spin on android was annoying me!
I couldn't see an easy way to prevent that, so have just made it a single iteration animation for all clients.

![infinite-spin](https://user-images.githubusercontent.com/9192912/223865006-f6cdab3f-28a1-4120-86f0-c2b5e5aca9e3.gif)